main_test.go

```diff
@@ -35,6 +38,17 @@ func TestSRIOV(t *testing.T) {
 	suite.Run(t, new(sriov.Suite))
 }
 
+func TestOVS(t *testing.T) {
+	ovsSuite := new(ovs.Suite)
+	ovsSuite.SetT(t)
+	// run only Kernel2Kernel test for now
+	ovsSuite.Run("TestKernel2Kernel", func() {
+		ovsSuite.SetupSuite()
+		defer ovsSuite.TearDownSuite()
+		ovsSuite.TestKernel2Kernel()
+	})
```

Comment:
For debug you can use a pkg `flag` and testify.m option.

Note: you can use a hack with pkg `flag`

```go
f := flag.Lookup("testify.m")
require.NoError(t, flag.Set("testify.m", "TestKernel2Kernel"))
defer func() { _= flag.Set("testify.m", f.Value)}()
suite.Run(t, new(ovs.Suite))
```

Reply:
@denis-tingaikin done, let me check test run now.